README.md

```diff
@@ -111,16 +106,13 @@ Users that contributed between December 1, 2020, 12:00 AM GMT-6 and December 15,
 
 For further reading see the [luxon time zone documentation](https://moment.github.io/luxon/docs/manual/zones) and the [List of tz values](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
 
--   The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. For example, in the example CSV above, `CSV_COLUMN_NUMBER_FOR_GITHUB_ID = 0` and `CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = 1`. If you choose not to use an alternate id, you can put the same column number in both `CSV_COLUMN_NUMBER_FOR_GITHUB_ID` and `CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID`.
-
--   To filter out events for which the author is the owner of the repository, simply set the `IGNORE_SELFOWNED_EVENTS` environment variable to "true", otherwise leave as `IGNORE_SELFOWNED_EVENTS = ""`.
-
 ### To run:
 
 In your terminal, type `cat {path/to/CSVfile}.csv | node index.js {date1} {date2}`
```

Comment:
Since we're making changes anyway, I've been meaning to mention for a while now that I find it easier to run starfish with:
`node index.js {date1} {date2} <{path/to/CSVfile}.csv`

(The dates can come before or after the redirection, but I find this sequence slightly clearer.)

Reply:
I actually find your way more confusing. 
Since the CSV is providing the input data, I'd expect it to be first (like an input). In this order, I expect that CSV to be the one we're writing to.
I'd also think it would be more confusing when redirecting the output to a new file because then there are 2 redirects going in opposite directions.
Can you explain why this way is clearer to you?

Reply:
I grew up using > and < all the time, and | very rarely. Introducing cat into the mix seems like adding complexity for no real benefit. (Especially since I grew up with `type` rather than `cat`, and always found the name `cat` to be less intuitive.) I think of it as "run index using these 2 dates, and instead of getting input from stdin, get it from this file". 

Hopefully before too long we will have starfish require you to pass an input file, which will remove the need to use < or |. At that point, either we would require command line switches to identify the different args (`node index.js --start {date1} --end {date2} --input {path/to/csv}`)  or we would lock a sequence in place, at which point I would agree that the input should come first (`node index.js {path/to/csv} {date1} {date2}`). 

Which is all to say that I feel moderately strongly that it would be better using > instead of cat |, and weakly about the sequencing. But if you want to leave it as it is for now, that's totally fine, and we can revisit everything when we stop using stdin.

Reply:
ok, that makes sense to me. 
I agree that `cat` is a stupid way to type something out so I'll switch it to your way, but I'm going to put the dates after the input file.
It looks as though putting a space after the < works just the same `node index.js < CSVsToParse/shortCSV.csv 2020-10-01 2020-11-01` and it looks cleaner to me. Is that ok, or is it typically used without the space?

Reply:
You are correct that it works with or without whitespace after the >. Personally I prefer no whitespace, because that clearly communicates to me that the > and the filename are closely related and tightly bound, as opposed to being 2 unrelated parameters. But I think I see others include a space at least as often as not, so having the space is fine.